src/components/Footer.tsx

```diff
@@ -0,0 +1,100 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo, SortType } from '../Types';
+import { client } from '../utils/client';
+
+interface Props {
+  numberOfActiveTodos: number,
+  selectedTab: SortType,
+  setSelectedTab: React.Dispatch<React.SetStateAction<SortType>>,
+  isThereCompletedTodos: boolean,
+  todo: Todo[],
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>,
+  setDeleteErrorMessage: React.Dispatch<React.SetStateAction<string>>,
+  setIsEveryThingDelete: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const Footer:React.FC<Props> = ({
+  numberOfActiveTodos,
+  selectedTab,
+  setSelectedTab,
+  isThereCompletedTodos,
+  todo,
```

Comment:
The prop name 'todo' should be changed to 'todos' for better readability and consistency.